packages/viewer/src/rendering/remote.ts

```diff
@@ -60,9 +57,9 @@ function requestFrame(api: StreamApi): RemoteRenderer {
       );
     });
 
-    return Async.timeout(timeout, update).finally(() =>
```

Comment:
@danschultz I would specifically appreciate your feedback here, as I believe these changes were put in somewhat intentionally, and I am now proposing this timeout be removed.

Reply:
@MadisonEhlers-Vertex what is the case for removing the timeout?

Reply:
Some of this was explained above. The main idea is that we don't want the promises to return at all if the client drops the frames. This was the root issue. Adding a timeout here will trigger the timeout on the event that the client dropped frames, even though the server is rightly closing the stream.

Reply:
This is very intentional. If removed, there's no way to complete the returned promise in the scenario that the client fails to receive a frame.